src/main/java/org/jenkinsci/plugins/github/config/GitHubPluginConfig.java

```diff
@@ -171,7 +171,7 @@ public boolean configure(StaplerRequest req, JSONObject json) throws FormExcepti
 
     @Override
     public String getDisplayName() {
-        return "GitHub Plugin Configuration";
+        return "GitHub Configuration";
```

Comment:
And yes, i know about help buttons, but they may be broken by core like in different plugins today.